syntax = "proto3";

// python3 -m grpc_tools.protoc --proto_path=. ./data.proto --python_out=../ --grpc_python_out=../

service GetDataService {
    rpc GetData (Posts) returns (CheckReponse) {}
}

service GetMetricService {
    rpc GetMetric (Metrics) returns (ServerReponse) {}
}

message Posts {
    string posts = 1;
}

message CheckReponse {
    bool received = 1;
}

message Metrics {
    string metrics = 1;
}

message ServerReponse {
    bool response = 1;
}